config/settings.yml

```diff
@@ -14,3 +14,100 @@ issue_manager:
   repo_names: []
 travis_event:
   enabled_repos: []
+trello:
+  # https://github.com/jeremytregunna/ruby-trello#configuration
+  oauth:
+    consumer_key:
+    consumer_secret:
+    oauth_token:
+    oauth_token_secret:
+  basic_auth:
+    developer_key:
+    member_token:
+  organization_id:
+  teams:
```

Comment:
These should be defined elsewhere during deployment (I'm sure you already knew that...just clarfiying)